src/components/Review.jsx

```diff
@@ -43,8 +64,24 @@ const Review = () => {
 					<img src={review.review_img_url} alt="" />
 					<h2>Posted by: {review.owner}</h2>
 					<p>{review.review_body}</p>
-					<ul className="review-card-icon">
-						<a href="/">{review.votes}👍</a>
+					<ul className="card-icon">
+						<button
+							onClick={(e) => {
+								handleReview(-1);
+							}}
+							disabled={review.votes < reviewTemp.votes}
+						>
+							⬇
+						</button>
+						<div className="vote-count">{review.votes}</div>
+						<button
+							onClick={(e) => {
+								handleReview(1);
+							}}
+							disabled={review.votes > reviewTemp.votes}
+						>
+							⬆
+						</button>
```

Comment:
As Mitch said before, it would be better to have the error message display under the button, rather than taking over the whole page. If the voting functionality isn't working, you don't want to remove the entire page for the user. They should still be able to see the reviews etc.